### PR TITLE
Add ES2015 support using Babel

### PR DESCRIPTION
**Pull request by ragingwind**
I made a new one to proof of concept for https://github.com/sindresorhus/generator-nm/issues/10 Please give me any ideas

- Copy source files to src under if babel is set
- Add ES2015 code with template in source files
- Add npm packages for babel
- Add scripts for babel, babel and prepublish
- No test yet

### CODE REVIEW

File: app/templates/_package.json [hunk 1]
```diff
@@ -14,7 +14,9 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "xo && ava"
+    "test": "<% if (babel) { %>npm run babel && <% } %>xo && ava"<% if (babel) { %>,
+    "babel": "babel src --out-dir ./ --presets es2015",
```
Comment (sindresorhus) [inline app/templates/_package.json]:
```
"babel": "babel src --out-dir . --presets=es2015",
```

**Comment by ragingwind** [comment]
done. I added test and update test script in order to xo test first

```
"test": "xo <% if (babel) { %>--ignore=*.js && npm run babel <% } %>&& ava"<% if (babel) { %>,
```

**Comment by jamestalmage** [comment]
What is blocking this? Just fixing the merge conflict?

**Comment by sindresorhus** [comment]
Oops, totally forgot about this. Yeah, seems good, just need to fix the merge conflict.

**Comment by ragingwind** [comment]
Ah.. me too. forgot this. I will update soon

**Pull request closed without merging**